tools/mk/Makefile.targ

```diff
@@ -122,16 +122,12 @@ check-bash: $(BASH_FILES:%=%.bashchk) $(BASH_FILES:%=%.bashstyle)
 %.bashstyle: %
 	$(BASHSTYLE) $^
 
-.PHONY: check-jscs
-check-jscs::
-	$(JSCS) $(JS_FILES)
-
 .PHONY: check-eslint
 check-eslint::
 	$(ESLINT) $(JS_FILES)
 
 .PHONY: check
-check: check-eslint check-jscs check-bash
+check: check-eslint check-bash
```

Comment:
Did you get a chance to add back the style target?

Reply:
Without the environment variables, it runs all of the rules both lint and style. Do you want to separate hem in the `Makefile.targ` like:  `check-lint` and `check-style`? I would keep `check-eslint` for the default behavior.

Reply:
Probably I should rename these: https://github.com/restify/node-restify/pull/1527/files#diff-b67911656ef5d18c4ae36cb6741b7965R52

Reply:
Could be something like this:

```
.PHONY: check-eslint
check-eslint::
	$(ESLINT) $(JS_FILES)

.PHONY: check-lint
check-lint::
	NO_STYLE=true $(ESLINT) $(JS_FILES)

.PHONY: check-style
check-style::
	NO_LINT=true $(ESLINT) $(JS_FILES)

.PHONY: check
check: check-lint check-style check-bash
	@echo check ok
```

Reply:
Yes, if we can keep two separate targets it'll keep the makefile as it was, but all on eslint which would be awesome.

Reply:
Fixed